layers/+lang/typescript/README.org

```diff
@@ -45,17 +48,32 @@ file.
 (setq-default dotspacemacs-configuration-layers '(typescript))
 #+END_SRC
 
-or if you don't need linting.
+If you don't need linting.
 
 #+BEGIN_SRC emacs-lisp
 (setq-default dotspacemacs-configuration-layers '(
-  (typescript :variables 
-              typescript-use-tslint nil))
+  (typescript :variables
+              typescript-use-tslint  nil))
+
+#+END_SRC
+
+If you need formatting on save.
```

Comment:
Use a `:` at the end of the line. The phrasing can also be a little bit improved, like «If you want formatting on save, you can use:».